src/controllers/signup.controller.js

```diff
@@ -0,0 +1,40 @@
+import SignupService from '../services/signup.service';
+import BcryptService from '../services/bcrypt.service';
+import Response from '../helpers/response';
+
+const response = new Response();
+/**
+ * @exports
+ * @class UsersController
+ */
+class UsersController {
+  /**
+ * users can create an account
+ * @static
+ * @description POST /api/v1/signUp
+ * @param {object} req request object
+ * @param {object} res response object
+ * @memberof UsersController
+ * @returns {object} data
+ */
+  static async signUp(req, res) {
+    const userExist = await SignupService.checkUserExistByEmail(req.body.email);
+    if (!userExist) {
```

Comment:
@higustave-ops , it's better if you move all if checks into middleware file instead